lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -1,10 +1,11 @@
 import { Descriptor, Format, InputDescriptorV1, InputDescriptorV2, PresentationSubmission, Rules, SubmissionRequirement } from '@sphereon/pex-models';
-import { IVerifiableCredential, WrappedVerifiableCredential } from '@sphereon/ssi-types';
+import { IVerifiableCredential, OriginalVerifiableCredential, WrappedVerifiableCredential } from '@sphereon/ssi-types';
 import jp from 'jsonpath';
+import _ from 'lodash';
```

Comment:
Not a fan of adding a dep for something small. This is a lib used by others, and as such should have minimal deps

Reply:
removed this dep